src/modules/logger/logger.cpp

```diff
@@ -609,15 +614,23 @@ void Logger::add_default_topics()
 	add_topic("vehicle_status");
 }
 
-void Logger::add_calibration_topics()
+void Logger::add_thermal_calibration_topics()
 {
 	// Note: try to avoid setting the interval where possible, as it increases RAM usage
-
 	add_topic("sensor_gyro", 100);
 	add_topic("sensor_accel", 100);
 	add_topic("sensor_baro", 100);
 }
 
+void Logger::add_system_identification_topics()
+{
+	// for system id need to log imu and controls at full rate
+	add_topic("sensor_gyro");
+	add_topic("sensor_accel");
+	add_topic("actuator_controls_0");
```

Comment:
@bkueng What happens here when common topics already added the topic. From initial testing it seems that it goes with whatever was set first. So this won't work, is that correct? Can we modify it so that if you add a topic twice, the rate from the second add is used?